src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -290,25 +308,27 @@ bool Profile::ShouldBeLayered(const Json::Value& json) const
 // <none>
 void Profile::LayerJson(const Json::Value& json)
 {
+    auto defaultAppearanceImpl = winrt::get_self<implementation::AppearanceConfig>(_DefaultAppearance);
+
     // Profile-specific Settings
     JsonUtils::GetValueForKey(json, NameKey, _Name);
     JsonUtils::GetValueForKey(json, GuidKey, _Guid);
     JsonUtils::GetValueForKey(json, HiddenKey, _Hidden);
     JsonUtils::GetValueForKey(json, SourceKey, _Source);
 
     // Core Settings
-    JsonUtils::GetValueForKey(json, ForegroundKey, _Foreground);
-    JsonUtils::GetValueForKey(json, BackgroundKey, _Background);
-    JsonUtils::GetValueForKey(json, SelectionBackgroundKey, _SelectionBackground);
-    JsonUtils::GetValueForKey(json, CursorColorKey, _CursorColor);
-    JsonUtils::GetValueForKey(json, ColorSchemeKey, _ColorSchemeName);
+    JsonUtils::GetValueForKey(json, ForegroundKey, defaultAppearanceImpl->_Foreground);

```

Comment:
Could we save a lot of duplicated code by just doing a 

```c++
defaultAppearanceImpl->LayerJson(json);
```

here? Then both Profile and Appeanace wouldn't need to know how to deserialize these properties?